### ISSUE TITLE
Using existing session

### ISSUE THREAD

**Issue by baratrion** [MachinePublishers/jBrowserDriver#225]
Maybe related to #94.

Is there a way I can re-use an existing session created with `JBrowserDriver`? Imagine a scenario, where I don't necessarily `.quit()` the browser, but would like to be able to re-use it with a new `JBrowserDriver` instance?

**Comment by event-horizon-acct** [comment]
This is JBrowserDriver.reset() but it has a memory leak, #204

**Comment by hollingsworthd** [comment]
This is JBrowserDriver.reset() but it has a memory leak, #204

**Comment by baratrion** [comment]
But I don't want to lose the browser state. Is this possible with `.reset()`?

**Comment by hollingsworthd** [comment]
So you want to "suspend" the instance basically? There's nothing for that right now. Best work around would be allocating lots of virtual ram, relying on OS paging, and increases the `instances` `Setting`

**Comment by baratrion** [comment]
Yes. So, I keep created instances in-memory for later access. Thanks.
